tests/c/tcp-ping-pong.c

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* This should come first. */
+#define _POSIX_C_SOURCE 200809L
+
+/*====================================================================================================================*
+ * Imports                                                                                                            *
+ *====================================================================================================================*/
+
+#include <arpa/inet.h>
+#include <assert.h>
+#include <demi/libos.h>
+#include <demi/sga.h>
+#include <demi/wait.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+
+/*====================================================================================================================*
+ * Constants                                                                                                          *
+ *====================================================================================================================*/
+
+/**
+ * @brief Data size.
+ */
+#define DATA_SIZE 64
```

Comment:
It would be nice if both DATA_SIZE and MAX_BYTES were just default values for their respective values, with the actual values being overridable via command line arguments.

Reply:
Sure, but the three other tests are currently relying on compile-time constants as well. May we create a good first issue for this?

Reply:
Okay.